airbyte-integrations/connectors/source-jira/source_jira/streams.py

```diff
@@ -92,6 +93,15 @@ def read_records(self, **kwargs) -> Iterable[Mapping[str, Any]]:
             if not (self.skip_http_status_codes and e.response.status_code in self.skip_http_status_codes):
                 raise e
 
```

Comment:
As someone with no context on this issue reading the code this looks like we're sweeping errors under the rug. Could you leave a source code comment explaining why we do this?

Reply:
that was no part of the original fix) i'll check and leave the comment what's going on there.

Reply:
I meant the lines you added :)

Reply:
Done.